widget/entry.go

```diff
@@ -84,13 +84,12 @@ func (e *entryRenderer) BackgroundColor() color.Color {
 }
 
 func (e *entryRenderer) Refresh() {
-	e.placeholder.refreshTextRenderer()
-	e.placeholder.Hide()
 	if e.text.len() == 0 && e.entry.Visible() {
 		e.placeholder.Show()
+	} else if e.placeholder.Visible() {
+		e.placeholder.Hide()
 	}
 
-	e.text.refreshTextRenderer()
```

Comment:
Is there no ramification or taking out this (and the other, above) renderer refresh calls?

Reply:
I didn't notice any.  The original debugging showed that `textProvider.Refresh()` was getting called 3 times for every input, and my thinking was if I got that down to 1 that would solve the problem.  It didn't, but seemed like the right approach to making it better regardless.

Reply:
Ok, thanks - just wanted to check